.github/actions/solaris/action.yml

```diff
@@ -0,0 +1,89 @@
+name: Solaris
+inputs:
+  configurationParameters:
+    default: ''
+    required: false
+  runExtraTests:
+    default: false
+    required: false
+runs:
+  using: composite
+  steps:
+    - name: Solaris
+      uses: vmactions/solaris-vm@v1
+      with:
+        release: "11.4-gcc"
+        usesh: true
+        copyback: false
+        # Temporarily disable sqlite, as FreeBSD ships it with disabled double quotes. We'll need to fix our tests.
```

Comment:
is it really the same for solaris ?

Reply:
No, you are right. I have removed the note and also configure options `--without-sqlite3` and `--without-pdo-sqlite`.